### PR TITLE
Homework 6 is done

### PR DESCRIPTION
**Pull request by hakob7475**

### CODE REVIEW

File: src/pizzaOrder/Order.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+    public Order(List<Pizza> items) {
+        orderId++;
+        customerId++;
+        checkPizzaName(items);
+        this.items = items;
+
+    }
+
+    private void checkPizzaName(List<Pizza> items) {
```
Comment (LevonAloyan) [inline src/pizzaOrder/Order.java:24]: This method does not belong to the Order class please move it to Pizza class

File: src/pizzaOrder/Order.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        for (int i = 0; i < items.size(); i++) {
+            if (items.get(i).getName().length() < 4 || items.get(i).getName().length() > 20) {
+                items.get(i).setName("customer_" + customerId + "_" + (i + 1));
+            }
+        }
+    }
+
+    public void displayingPizzaAttributes(Pizza pizza) {
```
Comment (LevonAloyan) [inline src/pizzaOrder/Order.java:32]: Please rename this method to `displayPizzaAttributes`

File: src/pizzaOrder/Order.java (hunk 1 above)
Comment (hakob7475) [reply src/pizzaOrder/Order.java]: Done

File: src/pizzaOrder/Order.java (hunk 2 above)
Comment (hakob7475) [reply src/pizzaOrder/Order.java]: Done

File: src/pizzaOrder/Pizza.java [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+            }
+        }
+        for (Ingredient ingredient : Ingredient.values()) {
+            if (ingredient.getName().equals(ingredientName)) {
+                ingredients.add(ingredient);
+            }
+        }
+    }
+
+    private void checkPizzaName(String name) {
```
Comment (LevonAloyan) [inline src/pizzaOrder/Pizza.java:46]: Now the second part, here you are violating the Single Responsibility principle, please create Separate class for this logic, like PizzaValidator and validate pizza name in that

Comment (hakob7475) [reply src/pizzaOrder/Pizza.java]: The method has been changed

**Pull request closed without merging**